CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 # Changelog
 
+## [1.1.0] - 2021-10-10
+
+### Added
+- SPIRE images are now published to GitHub Container Registry. They will continue to be published to Google Container Registry for the next release (#2576,#2580)
+- SPIRE Server now implements the TrustDomain API, which allows callers to manage dynamic federation relationships (#2497,#2524,#2532,#2533,#2535,#2536,#2541,#2544,#2545,#2546,#2547,#2549,#2550,#2557,#2559,#2560,#2561,#2562,#2563,#2564,#2565,#2566,#2568,#2571,#2572,#2574,#2575).
+- The TPM DevID Node Attestor emits a new `issuer:cn` selector with the common name of the issuing certificate (#2581)
+- The K8s Bundle Notifier plugin now supports pushing the bundle to resources in multiple clusters (#2531)
+- The SVIDStore plugin type has been introduced to enable, amongst other things, agentless workload scenarios (#2176,#2483)
+- A built-in AWS Secrets Manager SVIDStore plugin has been introduced, which can push workload SVIDs into AWS secrets (#2542)
```

Comment:
The real goal of this work is to enable agentless workloads, and specifically serverless workloads. IMO we should strongly point folks towards an agent where they can afford one. In alignment with that philosophy, the below recommendation suggests that we clarify the purpose of this plugin in an effort to reduce its general use in circumstances where an agent is the better option

```suggestion
- A built-in AWS Secrets Manager SVIDStore plugin has been introduced, which can push workload SVIDs into AWS secrets for use in Lambda functions etc.(#2542)
```

Reply:
Good suggestion